dev-tools/roxctl.sh

```diff
@@ -7,13 +7,16 @@ set -eo pipefail
 
 is_operator_on_openshift() {
   local result=0
-  kubectl get clusterversions.config.openshift.io version | grep -v "No resources found" > /dev/null
-  if [[ "$?" -ne "0" ]]; then
+  cmd() {
+      kubectl get clusterversions.config.openshift.io version | grep -v "No resources found" > /dev/null
```

Comment:
If we have a function we can check both conditions at once
```suggestion
      kubectl get clusterversions.config.openshift.io version | grep -v "No resources found" > /dev/null && kubectl get centrals.platform.stackrox.io -n stackrox | grep -v "No resources found" > /dev/null
```